documentation/decisions/adr/000N-video-conferencing.md

```diff
@@ -0,0 +1,166 @@
+- **Related Issue:** [#99](https://github.com/HHS/grants-equity/issues/99) <!-- RECOMMENDED -->
+- **Deciders:** Lucas Brown, Billy Daly, Sarah Knopp, Sumi Thaiveettil
+- **Tags:** communucations, open source, video conferencing
+
+## Context and Problem Statement  
+
+The project aims to implement a reliable video conference platform to facilitate seamless communication for both internal collaborations involving HHS and contractors, as well as external engagements with members of the public. The primary objective of this ADR is to evaluate and ultimately choose a suitable video conference tool that aligns with the project's requirements. The selected platform should enable efficient virtual meetings, ensuring clear and effective communication among project stakeholders, and accommodating diverse use cases, from internal team discussions to public outreach and engagement.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+#### Must Have
+
+- **Public Access**: If they have the right link, members of the public can join a video call without creating an account
+Waiting Room: Meeting organizers can configure their meetings to require that attendees must be admitted before they can join the call  
+- **Video Recording**: Meeting organizers can record a video call from within the platform  
+- **Screen Sharing**: Attendees can share their screen (if given the appropriate permissions by meeting organizer)  
+- **Chat**: Users can post comments and questions in a chat that are visible to other attendees  
+- **Phone Support**: Users can join by phone if they don't have access to a computer for video  
+- **Live Transcription**: The platform supports live transcription for attendees that may need closed captioning  
+- **Authority to Operate**: The platform should be covered under the existing ATO for Grants.gov 
+
+#### Nice to Have
+
+- **Webinar**: The platform supports a webinar format, i.e. attendees who can join and post questions but not see one another or unmute without permission  
+- **Breakout Rooms**: A meeting organizer can split users out into virtual "breakout rooms" for small-group discussions  
+- **Open Source**: The code to run this platform is open source and offers a self-hosting option  
+- **Attendance Tracking**: The platform allows host to access attendees to make follow up and attendance tracking easier
+- **Community & Support**: The platform has a strong community for ongoing support, updates, and bug fixes
+- **Scalability**: The platform should be able to hadndle a growing number of participants and meetings wihtout performance issues
+
+
+## Options Considered
+
+- [Zoom](https://zoom.us/) - *decision* because of  
+- [Microsoft Teams](https://www.microsoft.com/en-us/microsoft-teams/group-chat-software) - *decision* because of  
+- [Google Meet](https://meet.google.com/) - *decision* because of  
+- [Jitsu](https://meet.jit.si/) - *decision* because of  
+
+## Decision Outcome <!-- REQUIRED -->
+
+TBD 
```

Comment:
Are we waiting on more information, or just general discussion?

Reply:
Sorry, my fault. I created 2 PRs. N00b over here. The actual PR to review is this one: https://github.com/HHS/grants-equity/pull/337.